src/openforms/utils/date.py

```diff
@@ -31,3 +35,19 @@ def parse_date(value: str) -> date:
     dt = datetime.fromisoformat(value)
     assert dt.tzinfo is not None, "Expected the input variable to be timezone aware!"
     return timezone.localtime(value=dt).date()
+
+
+def parse_datetime(value: str) -> None | datetime:
+    try:
+        return _parse_datetime(value)
+    except ValueError:
+        logger.info("Can't parse datetime '%s', using empty string.", value)
```

Comment:
the "using empty string" part is confusing, because you actually return `None`, and this function itself does not know how the caller will handle this `None` value.

```suggestion
        logger.info("Can't parse datetime '%s', falling back to 'None' instead.", value)
```